app/lists.rb

```diff
@@ -0,0 +1,73 @@
+module Listing
+	extend Discordrb::Commands::CommandContainer
+	
+	wing_list = Array.new
+	#The array that is wing_list is created.
+	
+#  def self.find_role(event, role)
+#    has_role = false
+#    event.server.roles.each do |server_role|
+#      next unless %w(botadmin wingcaptain).any?(server_role.name)
+#      has_role = true
+#    end
+#    return role
+#  end
+
+  def self.role?(event, role)
+    has_role = false
+    event.server.roles.each do |server_role|
+      next unless server_role.name == role
+      has_role = true
+      break
+    end
+    return has_role
+  end
+	
+  command :winglist, description: "Lists those who are requesting wing." do |event|
+  #Simple, return the array's contents. Also, admin lock this.
+    break unless event.user.role?(find_role(event, 'wingcaptain'))
+    #Check to see if they have the "wingcaptain role"
+    event << wing_list.take(wing_list.length)
+	#Why do I have the feeling this will barf data unintelligibly?
+  end
+  
+  command :wingremove, description: "Remove a person from the wing queue", min_args: 1, max_args: 1 do |event, target|
+  #Remove someone who is on the wing list. Defo admin lock.
+    break unless event.user.role?(find_role(event, 'wingcaptain'))
+	#Check to see if they have the "wingcaptain role". I am aware this is a double comment.
+	wing_list.delete(target)
+	event.respond "Removed" + target + "from the queue."
```

Comment:
You use both string concatenation like this, and variable injection elsewhere. I'd prefer `"Removed #{target} from the queue."` over concatenation for legibility.